client/components/gravatar/docs/example.jsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gravatar from 'components/gravatar';
+
+export default React.createClass( {
+
+	displayName: 'Gravatar',
+
+	render() {
+		const user = {
```

Comment:
As much as I'm sure everyone would have enjoyed admiring @rralian's photo, that's a good shout @mtias. I'll change it now.